packages/dashboard-core-plugins/src/events/TabEvent.ts

```diff
@@ -1,15 +1,6 @@
-class TabEvent {
-  static focus = 'TabEvent.focus';
-
-  static blur = 'TabEvent.blur';
-
-  static openPQObject = 'TabEvent.openPQObject';
-
-  static openControl = 'TabEvent.openControl';
-
-  static reload = 'TabEvent.reload';
-
-  static clearAllFilters = 'TabEvent.clearAllFilters';
-}
+const TabEvent = {
+  focus: 'TabEvent.focus',
+  blur: 'TabEvent.blur',
+} as const;
```

Comment:
Instead use `Object.freeze`:
```suggestion
const TabEvent = Object.freeze({
  focus: 'TabEvent.focus',
  blur: 'TabEvent.blur',
});
```
![image](https://github.com/user-attachments/assets/0e4dd96c-f4bc-45d0-9a4e-ffaf09ade16c)
It's effectively the same thing, and also ensure the object cannot be modified.